static/bidder-info/taboola.yaml

```diff
@@ -1,15 +1,20 @@
-endpoint: "https://display.bidder.taboola.com/OpenRTB/TaboolaHB/auction"
+endpoint: "https://{{.MediaType}}.bidder.taboola.com/OpenRTB/PS/auction/{{.Host}}/{{.PublisherID}}"
 maintainer:
   email:  ps-team@taboola.com
 gvlVendorID: 42
 capabilities:
   app:
     mediaTypes:
       - banner
+      - native
   site:
     mediaTypes:
       - banner
+      - native
 userSync:
   redirect:
-    url: https://trc.taboola.com/sg/ps/1/cm?gdpr={{.GDPR}}&consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirect={{.RedirectURL}}
+    url: https://trc.taboola.com/sg/ps/1/cm?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirect={{.RedirectURL}}
+    userMacro: "<TUID>"
+  iframe:
+    url: https://cdn.taboola.com/scripts/ps-sync.html?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirect={{.RedirectURL}}
```

Comment:
I believe this should be related to discussion done [here](https://github.com/prebid/prebid-server/pull/2522/files#r1086900875).

redirect urls are not redirecting.

<img width="1792" alt="Screenshot 2023-02-08 at 9 16 03 PM" src="https://user-images.githubusercontent.com/24757781/217579275-3e211173-2e7a-474b-b22c-2dbf420128e6.png">
<img width="1792" alt="Screenshot 2023-02-08 at 9 15 42 PM" src="https://user-images.githubusercontent.com/24757781/217579344-9591c26e-a4e8-4a3c-848b-0673683c51f5.png">